backend/resources/projects.py

```diff
@@ -161,6 +161,8 @@ def projects_post(session, user_inst: User):  # pylint:disable=unused-argument,
 
     if None in [name, goal_raw, until_raw, id_institution, description, short]:
         return jsonify({'error': 'Missing parameter'}), 403
+    if milestones == "[]":
+        return jsonify({'error': 'Missing milestone'}), 403
```

Comment:
warum nicht in Zeile 206 (bzw. vorher 204) nach dem milestones_json = json.loads(..)
```
if len(milestones_json) == 0:
    return jsonify({'error': 'Missing milestone'}), 403
```

Das wär doch sauberer oder?